packages/xchain-binance/src/client.ts

```diff
@@ -244,17 +244,26 @@ class Client implements BinanceClient, XChainClient {
     try {
       const balances: BinanceBalances = await this.bncClient.getBalance(address)
 
-      return balances
-        .map((balance) => {
-          return {
-            asset: assetFromString(`${BNBChain}.${balance.symbol}`) || AssetBNB,
-            amount: assetToBase(assetAmount(balance.free, 8)),
-          }
-        })
-        .filter(
-          (balance) =>
-            !assets || assets.filter((asset) => assetToString(balance.asset) === assetToString(asset)).length,
-        )
+      let assetBalances = balances.map((balance) => {
+        return {
+          asset: assetFromString(`${BNBChain}.${balance.symbol}`) || AssetBNB,
+          amount: assetToBase(assetAmount(balance.free, 8)),
+        }
+      })
+
+      // make sure we always have the bnb asset as balance in the array
+      if (assetBalances.length === 0) {
+        assetBalances = [
+          {
+            asset: AssetBNB,
+            amount: assetToBase(assetAmount('0.0', 8)),
```

Comment:
Could you add `const BNB_DECIMAL = 8` to 'xchainjs-lib/packages/xchain-binance/src/utils.ts` (similar to other packages) to use it for example as follow here ^:

`assetToBase(assetAmount('0.0', 8))` -> `baseAmount(0, BNB_DECIMAL)`